docs/tasks/configure-pod-container/environment-variable-expose-pod-information.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Exposing Pod Information Through Environment Variables
```

Comment:
This is a little bit vague on who the information is being exposed to.  WDYT of "Exposing Pod Information To Containers" or "Accessing Pod Information From Containers".